config/application.rb

```diff
@@ -32,6 +32,24 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
+
+    config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+      address: 'smtp.mandrillapp.com',
+      port: '587',
+      domain: 'vagabonder.herokuapp.com',
+      user_name: ENV["MANDRILL_NAME"],
+      password: ENV["MANDRILL_KEY"],
+      authentication: 'plain',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.